### ISSUE TITLE
access token or refresh token will be reset when another is set

### ISSUE THREAD

**Issue by Lexachoc** [keycloak/keycloak#35429]
### Before reporting an issue

- [X] I have read and understood the above terms for submitting issues, and I understand that my issue may be closed without action if I do not follow them.

### Area

account/api

### Describe the bug

When I set the `SSO Session Settings` :
![image](https://github.com/user-attachments/assets/7a535d94-f978-45e5-adcf-9c09e303d2ce)

click "Save".

I got:
```json
{
    "access_token": "...",
    "expires_in": 300,
    "refresh_expires_in": 360,
    "refresh_token": "...",
    "token_type": "Bearer",
    "id_token": "...",
    "not-before-policy": 0,
    "session_state": "...",
    "scope": "openid email profile"
}
```
when make a POST request to `http://localhost:8080/realms/{real_name}/protocol/openid-connect/token`

Now the `refresh_expires_in` expires in 360s this is what I've set (6 minutes). And the access token expires in 300s (`expires_in`)

and then I go the Tokens Tab:
![image](https://github.com/user-attachments/assets/41f0eeaf-f541-40f2-a8e4-25b1b89a6e46)

And change the Access Token Lifespan to 1 minute.
click "Save".

make a POST request again. I got:
{
    "access_token": "..",
    "expires_in": 60,
    "refresh_expires_in": 1800,
    "refresh_token": "...",
    "token_type": "Bearer",
    "id_token": "...",
    "not-before-policy": 0,
    "session_state": "...",
    "scope": "openid email profile"
}

Now, the `expires_in` becomes 60s, this is correct. BUT the refresh token expire time (`refresh_expires_in`) becomes 1800s!

And if I go back to the Sessions Tab and click save again, the `refresh_expires_in` becomes 360 again. BUT the access token becomes to 300s!

Is this by design or a bug? It's pretty confusing and no idea why it's happening.

### Version

25.0.4

### Regression

- [ ] The issue is a regression

### Expected behavior

I expect that the access token and refresh token expire time can be set according to the account UI settings.

### Actual behavior

The access token (refresh token) expiration times is reset when I set a value for refresh token (access token).

### How to Reproduce?

Go to the account UI, and go to `Ream Settings` -> `Sessions` and `Tokens` tabs to set the value accordingly.

**Comment by Lexachoc** [comment]
Hi, I tried to redo this again and suddently it works as expected...

Absolutely so idea why.

Now I got:

```json
{
    "access_token": "...",
    "expires_in": 120,
    "refresh_expires_in": 360,
    "refresh_token": "...",
    "token_type": "Bearer",
    "id_token": "...",
    "not-before-policy": 0,
    "session_state": "...",
    "scope": "openid email profile"
}
```

It seems that the account UI displayed a toast message `Realm updated successfully` to me, but in fact something was wrong and after a while when I tried again, it really worked.

**Comment by mposolda** [comment]
Based on the message `Realm updated successfully`, I guess that you are talking about admin console, rather than account console? The UI where you are updating realm is admin console.

Setting this to UI team to triage (if it is a valid issue).

**Comment by Lexachoc** [comment]
> Based on the message `Realm updated successfully`, I guess that you are talking about admin console, rather than account console? The UI where you are updating realm is admin console.
> 
> Setting this to UI team to triage (if it is a valid issue).

Yes, thank you for the correction here. It is **admin console** indeed!

**Comment by ssilvert** [comment]
@Lexachoc Sounds like it works properly sometimes? Do you have a way to consistently recreate the problem?

**Comment by Lexachoc** [comment]
@ssilvert After a few tries, I finally know why this happens!

The original settings:
![image](https://github.com/user-attachments/assets/0f9e7bf3-edb2-45e5-98fe-b94c2f1280b6)
![image](https://github.com/user-attachments/assets/cdd19b8e-e83c-4501-9961-f43690f426ab)
![image](https://github.com/user-attachments/assets/08db4297-ada0-40b5-9be7-8881b2fc7b0f)


To reproduce:
First, go to the `Sessions` tab, and change `SSO Session Idle` value from `2` to `5`, and save it.
![image](https://github.com/user-attachments/assets/4c47e77b-8fa5-4b4f-b607-f6af2c7f8411)

Second, go to the `Tokens` tab, and change `Access Token Lifespan` value from `1` to `2`, and save it.
![image](https://github.com/user-attachments/assets/194455b3-ea4e-4f57-a317-a161f1153243)

Check the response now:
See that the refresh token time is revert back to 2 mins (120s)!
![image](https://github.com/user-attachments/assets/5d85f911-558c-46ef-96d9-7a17c0d041ae)

And if I switch back to the `Sessions` tab to confirm:
it is 5mins!
![image](https://github.com/user-attachments/assets/98db7c22-3077-4e55-9bfa-ec416aa1c4b3)

But it I reload the page:
it becomes 2mins! As I saw from the response
![image](https://github.com/user-attachments/assets/4a0b8211-98aa-48e1-be7a-549fb43da30c)

**Comment by ssilvert** [comment]
Thanks for doing the research @Lexachoc
